Routing/ThriftRoutingLoader.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Overblog\ThriftBundle\Routing;
 
+use App\Kernel;
```

Comment:
wrong class replace by `Symfony\Component\HttpKernel\Kernel`

Reply:
fixing it right now.. thanks